CMakeLists.txt

```diff
@@ -59,7 +48,7 @@ endif()
 
 add_test(
     NAME pytest-main
-    COMMAND ${Python3_EXECUTABLE} -m pytest --junitxml=${CMAKE_BINARY_DIR}/junit-pytest-main.xml ${CMAKE_SOURCE_DIR}/tests
+    COMMAND ${Python_EXECUTABLE} -m pytest --junitxml=${CMAKE_BINARY_DIR}/junit-pytest-main.xml ${CMAKE_SOURCE_DIR}/tests
```

Comment:
With the removal at line 10, where is 'Python_EXECUTABLE' set now?

Reply:
It is set by the FindPython module. See: [documentation](https://cmake.org/cmake/help/latest/module/FindPython.html).

Reply:
ACK